lib/bindgen_gen.rs

```diff
@@ -7,7 +7,7 @@ const ALLOW_LIST_TYPE: &str =
     ".*ExternalBuffers.*|.*PRIME.*|.*MPEG2.*|.*VP8.*|.*VP9.*|.*H264.*|.*HEVC.*|\
     .*JPEG.*|VACodedBufferSegment|.*AV1.*|VAEncMisc.*|VASurfaceDecodeMBErrors|\
     VADecodeErrorType|.*VAProc.*|\
-    VACenc.*|VA_TEE_.*|VAEncryption.*|VA_PROTECTED_.*";
+    VACenc.*|VA_TEE_.*|VAEncryption.*|VA_PROTECTED_.*|VAEncPacked.*";
```

Comment:
My bad...  I fold this line for the intel specific protected content headers.
Would you append the new value, VAEncPacked, to line.9?